features/delete_reference.feature

```diff
@@ -0,0 +1,11 @@
+#language: fr
+
+Fonctionnalité: Supprimer les références existantes
+
+  Scénario: Supprimer la référence à droite
```

Comment:
This scenario will only work if you are the author of the reference.

See the edition scenarios for examples (e.g. https://github.com/Hypertopic/HyperGlosae/blob/main/features/edit_glose_content.feature).